src/Helmet.js

```diff
@@ -276,9 +279,9 @@ const generateTagsAsString = (type, tags) => {
             })
             .join(" ").trim();
 
-        const innerHTML = tag.innerHTML || "";
+        const tagContent = tag.innerHTML || tag.cssText || "";
 
-        return `<${type} ${HELMET_ATTRIBUTE}="true" ${attributeHtml}${type === TAG_NAMES.SCRIPT ? `>${innerHTML}</${type}>` : `/>`}`;
+        return `<${type} ${HELMET_ATTRIBUTE}="true" ${attributeHtml}${type === TAG_NAMES.SCRIPT ? `>${tagContent}</${type}>` : `/>`}`;
```

Comment:
This is only checking for `TAG_NAMES.SCRIPT`, do you need to add `TAG_NAMES.STYLE`?